_includes/header.html

```diff
@@ -1,23 +1,78 @@
-    <img src="https://static.opensuse.org/chameleon/dist/images/logo/logo-white.svg" class="mr-2" alt="Logo">
-    <span>News</span>
+    <svg class="icon icon-2x mr-2">
+      <use xlink:href="#opensuse"></use>
+    </svg>
+    <span class="navbar-title">News</span>
   </a>
+
   <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#site-menu">
-    <span class="navbar-toggler-icon"></span>
+    <svg class="icon">
+      <use xlink:href="#menu-line"></use>
+    </svg>
   </button>
+
   <div class="collapse navbar-collapse" id="site-menu">
-    <div class="nav navbar-nav ml-auto">
-      <div class="nav-item">
-        <a class="nav-link" href="{{ "/feed.xml" | prepend: site.baseurl }}">
-          <span class="fa fa-rss"></span>
-          <span class="d-md-none">RSS Feed</span>
+    <ul class="nav navbar-nav mr-md-3">
+      <!-- Categories Dropdown -->
+      <li class="nav-item dropdown">
+        <a class="nav-link dropdown-toggle" href="#" id="cat-menu-link" role="button" data-toggle="dropdown"
+          aria-haspopup="true" aria-expanded="false">
+          <svg class="icon">
+            <use xlink:href="#newspaper-line"></use>
+          </svg>
+          Categories
         </a>
+        <div class="dropdown-menu" aria-labelledby="cat-menu-link">
+          {% for category in site.categories %}
+          {% capture category_name %}{{ category | first }}{% endcapture %}
+          <a class="dropdown-item" href="/category/{{ category_name | slugify }}">
+            {{ category_name }}
+          </a>
+          {% endfor %}
+        </div>
+      </li>
+
+      <li class="nav-item">
+        <a class="nav-link" href="https://github.com/openSUSE/news-o-o#how-to-submit-a-new-post">
+          <svg class="icon">
+            <use xlink:href="#quill-pen-line"></use>
+          </svg>
+          New Post
+        </a>
+      </li>
+
+      <li class="nav-item">
+        <a class="nav-link" href="{{ '/feed.xml' | prepend: site.baseurl }}">
+          <svg class="icon">
+            <use xlink:href="#rss-line"></use>
+          </svg>
+          RSS Feed
+        </a>
+      </li>
+    </ul>
+
+    <!-- DuckDuckGo Site Search -->
+    <form id="ddg-form" class="form-inline" action="https://duckduckgo.com/">
+      <div class="input-group">
+        <input id="ddg-input" class="form-control" type="search" name="q" placeholder="Search" />
+        <div class="input-group-append">
+          <button class="btn btn-secondary" type="submit">
+            <svg class="icon">
+              <use xlink:href="#search-line"></use>
+            </svg>
+          </button>
+        </div>
       </div>
-      <form class="form-inline" target="_blank" action="https://github.com/openSUSE/news-o-o/search" method="GET">
-        <input id="searchbox" class="form-control" type="text" name="q" placeholder="Search" lang="en">
-        <input type="hidden" value="Markdown" name="l">
-      </form>
-    </div>
+      <script>
```

Comment:
I don't see why this requires javascript

Reply:
That is to prepend query string `site:news.opensuse.org ` to the search keywords. So it can limit the search result to news.opensuse.org instead of the whole internet.

Reply:
Then why use ddg in the first place then

Reply:
We can of course use other search engine. I feel DDG is more privacy friendly than Google/Bing.

Reply:
Using github search works very well for indexing what matters though, I don't understand why it really needs a replacement (and when you click on the result, it loads it as html document from our markdown)

Reply:
The advantage of search engine is ordering the result. For example, when I search "tumbleweed plasma" to check if here are new plasma released.

The [GitHub search result](https://github.com/openSUSE/news-o-o/search?q=tumbleweed+plasma&unscoped_q=tumbleweed+plasma) doesn't care about dates of posts. (posts from 2015, 2016, 2017 are on the top of search results)

![image](https://user-images.githubusercontent.com/5836790/75609464-7264c600-5b11-11ea-8e33-da212bba8a25.png)

But [DDG search result](https://duckduckgo.com/?q=tumbleweed+plasma+site%3Anews.opensuse.org&t=ffab&ia=web) does.

![image](https://user-images.githubusercontent.com/5836790/75609463-68db5e00-5b11-11ea-9bcc-b031a68c349e.png)

Reply:
Very easy to fix for that matter, add `<input type="hidden" name="s" value="indexed">` and it will sort by date

Reply:
I can revert to GitHub form. But the solution is still let users leave news website and start to browsing GitHub... GitHub provides search APIs. You can embed its search result inside news website, search content without leaving the site.